esp-wifi/src/wifi/os_adapter.rs

```diff
@@ -1652,7 +1660,12 @@ pub unsafe extern "C" fn wifi_create_queue(
     queue_len: crate::binary::c_types::c_int,
     item_size: crate::binary::c_types::c_int,
 ) -> *mut crate::binary::c_types::c_void {
-    create_wifi_queue(queue_len, item_size)
+    static mut QUEUE_HANDLE: *mut crate::binary::c_types::c_void = core::ptr::null_mut();
```

Comment:
Does this need to be stored in a static? If the queue is `malloc`'d it should live until the driver calls `free`, right? So the pointer should always be valid. It doesn't look like `QUEUE_HANDLE` is accessed outside of this function.

Reply:
This function actually returns a pointer to a pointer to the queue while `queue_create` just returns a pointer to the queue

Given there is only one wifi-queue I thought this is easiest - but probably worth to double check `wifi_delete_queue`